Unite/Unite/Controllers/ChatController.cs

```diff
@@ -29,4 +29,11 @@ public ChatController(AssistantService assistantService)
         
         return Ok(response);
     }
+
+    [HttpGet("health")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public IActionResult HealthCheck()
+    {
+        return Ok();

```

Comment:
Hi @jasonpawar-infotrack,

Could you please provide constructive feedback to help improve the implementation?

Thank you!